codegen/scala-gen/src/test/scala/com/akkaserverless/codegen/scalasdk/ValueEntityTestKitGeneratorSuite.scala

```diff
@@ -164,6 +164,61 @@ class ValueEntityTestKitGeneratorSuite extends munit.FunSuite {
     assertNoDiff(sourceCode, expected)
   }
 
+  test("it can generate an specific integrationtest stub for the entity") {
+    val entity = createShoppingCartEntity()
+    val service = createShoppingCartService(entity)
+    val main = FullyQualifiedName("Main", packageNaming)
+
+    assertEquals(
+      ValueEntityTestKitGenerator.integrationTest(main, entity, service).content,
+      """package com.example.shoppingcart.api
+
+import com.akkaserverless.scalasdk.testkit.AkkaServerlessTestkit
```

Comment:
Are we moving out of using the `|`? I have see that at least one other suite were not using it. 

I don't mind. Maybe be even easier to read. Soon I will be write codegen for the replicated entity. I will use that new style then.

Reply:
Oh, this was not particularly intentional, I'm fine with either

Reply:
Maybe be easier if we remove the `|`.

Reply:
since we have mostly used `|` I think we should continue with that, and at least to me it reads a bit weird to suddenly have code to the very left